students/SLammertink/Lesson03/mailroom_1.py

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/env python3
+# Author: SLammertink
+# UW Self Paced Lesson 03
+# Mailroom part 1
+
+
+# Initiate the lists used
+don_list= [75, 25346.25, 125, 25, 200.50] # list with the donations
+name_list = ['Sukhmani Travers', 'Sebastien Mayo', 'Aryan Davila', 'Zayan Langley', 'Charlotte Bates'] #list with the donor names
+count_list =  [1, 2, 1, 3, 2] # list with a count of the donations given
+choices_list = ['Sending a Thank You', 'Creating a Report', 'quit'] # list from where user can chose
+
+# The Functions used:
+
+def name_in_list():
+    #Checks whether is name is in the name list if not it adds it
+    input_name = input("Please enter the donor's name: ")
+    if input_name not in name_list:
+        print('')
+        print("That name is not in the donor list, will add it now.") # notify the user that the name will be added to the list
+        print('')
+        name_list.append(input_name)
```

Comment:
Something unexpected happening here.  New donors aren't being added as expected.  Trying to run a report after adding a donor causes an error.  Take a look at exception handling.